.github/workflows/build-native.yml

```diff
@@ -2,28 +2,37 @@ name: Build Native Images
 on:
   release:
     types: [published]
+  pull_request:
```

Comment:
Do we need this empty line any more?

Reply:
Scoped the pull_request trigger to PRs targeting the "main" branch

Reply:
ok!